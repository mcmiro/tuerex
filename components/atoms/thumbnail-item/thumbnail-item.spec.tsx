import * as React from "react";
import {screen, render} from "@testing-library/react";
import ThumbnailItem from "./";

describe("Test ThumbnailItem component", () => {
	it("Should pass when image url is defined", () => {
		render(<ThumbnailItem imageUrl="/test" data-testid="thumbnail-item-test" />);

		const thumbnailItem = screen.queryByTestId("thumbnail-item-test");
		expect(thumbnailItem).toBeDefined();
		expect(thumbnailItem).toMatchSnapshot();
	});
});
