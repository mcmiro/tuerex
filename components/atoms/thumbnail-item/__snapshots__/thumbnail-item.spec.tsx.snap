// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test ThumbnailItem component Should pass when image url is defined 1`] = `null`;
